include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
> fi_log.h is user-visible. It's just in this case that the user is an external provider. The config.h file really shouldn't be included from fi_log.h.

I see the confusion here now, Jeff and I both thought that `fi_log.h` was internal only.